
body_to_frontright_fisheye:
  Matrix:
    - [-0.420, -0.484,  0.768,  0.385]
    - [-0.008,  0.848,  0.530, -0.038]
    - [-0.907,  0.217, -0.360, -0.050]
    - [ 0.000,  0.000,  0.000,  1.000]

body_to_frontright:
  Matrix:
    - [-0.417, -0.479,  0.773,  0.416]
    - [-0.007,  0.852,  0.524, -0.038]
    - [-0.909,  0.213, -0.358,  0.022]
    - [ 0.000,  0.000,  0.000,  1.000]

feet_center_to_frontright_fisheye:
  Matrix:
    - [-0.387, -0.492,  0.780,  0.323]
    - [-0.007,  0.847,  0.531, -0.034]
    - [-0.922,  0.200, -0.332,  0.548]
    - [ 0.000,  0.000,  0.000,  1.000]

feet_center_to_frontright:
  Matrix:
    - [-0.381, -0.487,  0.786,  0.356]
    - [-0.006,  0.851,  0.525, -0.036]
    - [-0.925,  0.195, -0.327,  0.617]
    - [ 0.000,  0.000,  0.000,  1.000]


